javascript/new-src/src/js/.jshintrc

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
If this is the same as the other one, is the system not clever enough to inherit? If not, can we have a symlink please?